bootstrap/bootstrap.go

```diff
@@ -96,26 +96,6 @@ func (b *Bootstrap) Start() int {
 	return exitStatus
 }
 
-// executeScript executes a script in a Shell, but the target is an interpreted script
```

Comment:
I moved this to the Shell, as it's shell related and a pretty common thing to want to do.